utils/errors.ts

```diff
@@ -0,0 +1,9 @@
+export function errorFilter(value: unknown | Error): boolean {
+  if (value instanceof Error) {
+    // TODO: Have a proper error reporting mechanism
+    console.log(`Errored at`);
+    console.log(value);
+    return false;
+  }
+  return true;
+}
```

Comment:
Do we expose the errors at all to the user?

Reply:
Nope